Formula/haxe.rb

```diff
@@ -16,6 +16,13 @@ class Haxe < Formula
   head do
     url "https://github.com/HaxeFoundation/haxe.git", :branch => "development"
     depends_on "opam" => :build
+
+    # ptmap 2.0.2 is needed for OCaml 4.05.0 compatibility
+    # See https://github.com/backtracking/ptmap/pull/4
+    resource "ptmap" do
+      url "https://github.com/ocaml/opam-repository/pull/10170.diff?full_index=1"
+      sha256 "2d5394638c370e654ee1b87e23a348d512da717df9bd52a16c1816c6047ba4f7"
```

Comment:
@ilovezfs it looks like the sha256 of 10170.diff is wrong.
When running `brew install haxe --HEAD` I get the following error:

```
==> Downloading https://github.com/ocaml/opam-repository/pull/10170.diff?full_index=1
==> Downloading from https://patch-diff.githubusercontent.com/raw/ocaml/opam-repository/pull/10170.diff?full_index=1
######################################################################## 100,0%
Error: SHA256 mismatch
Expected: 2d5394638c370e654ee1b87e23a348d512da717df9bd52a16c1816c6047ba4f7
Actual: 8b4938166ef02e3546898fb9d5742e466e7cb75f0a9a5548a999fbe38f504628
Archive: /Users/tanis/Library/Caches/Homebrew/haxe--ptmap-10170.diff
To retry an incomplete download, remove the file above.
```

Reply:
Yeah the PR was just updated. I will fix it :)

Reply:
@tanis2000 it should be fixed now via https://github.com/Homebrew/homebrew-core/pull/17316.

```
brew update
rm /Users/tanis/Library/Caches/Homebrew/haxe--ptmap-10170.diff
```
and then try again.

Reply:
It's working fine now. Thanks!

Reply:
You're welcome!